### DISCUSSION TITLE
Simple backend with nocodb [Jun 23]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2522] [Discord Threads]
> 🙋‍♂️ **heebers**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Yes, you can build a simple backend. Either you can use an existing database or you can use ours (data will be stored in database specifeid by NC_DB env var). Yes it does handle scaling up of APIs. It is free & open source and you will have to deploy it on your servers  
#  
> 🙋‍♂️ **heebers**  
#### Got it, thanks!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/989255282519584919)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
